README.md

```diff
@@ -130,7 +132,7 @@ security:
 # ...
 ```
 
-### Step 3 (Symfony 5.2-)
+### Step 3 (Symfony 4.4)
```

Comment:
A little below this point there is still a reference to Symfony 3:

```
**Symfony 3 Version:**
```yaml
# app/config/routing.yml
gesdinet_jwt_refresh_token:
    path:     /api/token/refresh
    defaults: { _controller: gesdinet.jwtrefreshtoken:refresh }
# ...
```

Reply:
I've done a once over on the README.